test/test_view_ops.py

```diff
@@ -475,6 +474,8 @@ def test_select_view(self, device) -> None:
         v[0] = 0
         self.assertEqual(t[2, 0], v[0])
 
+    # Lazy hasn't implemented unbind yet.
+    @onlyNativeDeviceTypes
```

Comment:
So, locally I was able to get this entire file to pass for LTC (before this PR, it had ~20 failures). I think we should probably opt LazyTensor into automatically running these tests, but that might take some work. The way to do it is by adding "lazy" to the list of devices used in `common_device_type.py`. I think that will automatically opt LTC into **all** of the op info tests though, and LTC will probably fail a bunch of those. Maybe we should either get LTC to opt just into this file somehow, or someone should painstakingly go through all of the OpInfo failures and xfail/fix them.

Reply:
You can do `instantiate_device_type_tests` in this file with `only_for` argument that includes LTC.

Reply:
ah nice :) Will do

Reply:
i guess that change would come later?  the `@onlyNativeDeviceTypes` macro means not lazy, for now?

Reply:
I could either stick that in this PR, or do it later (it sounds like it'll just be a one-liner to have this test file starting running in CI with LTC).